.github/workflows/release-rollback.yml

```diff
@@ -47,3 +47,30 @@ jobs:
           node-version-file: '.nvmrc'
           registry-url: 'https://wombat-dressing-room.appspot.com'
           scope: '@google'
+
+      - name: 'Get Current Release Version'
+        shell: 'bash'
```

Comment:
Avoid using `$GITHUB_ENV`, it's too magical.

- Give this an `id` (consider `release_version`)
- Write to `$GITHUB_OUTPUT`
- Then in your usage you can do `${{steps.release_version.outputs.RELEASE_VERSION}}`

I find this much better because it's very explicit what's happening.  Futzing around with the env feels to magical. Magical is bad.